app/utils/SessionKeys.scala

```diff
@@ -34,7 +34,7 @@ object SessionKeys {
   val whenDidTechnologyIssuesEnd = "whenDidTechnologyIssuesEnd"
   val wasHospitalStayRequired = "wasHospitalStayRequired"
   val whenHealthIssueHappened = "whenHealthIssueHappened"
-  //TODO: use when implementing 'yes' journey for hospital stay
+  val whenHospitalStayBegin = "whenHospitalStayStartedBegin"
```

Comment:
I think `whenHealthIssueStarted` rather than `whenHospitalStayBegin`  should be used. This is because it is used on the CYA page and payload builder. So `whenHospitalStayBegin` is not needed.